libs/jsonforms-components/src/lib/Controls/FormStepper/RenderStepElements.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  Categorization,
+  Category,
+  JsonFormsCellRendererRegistryEntry,
+  JsonFormsRendererRegistryEntry,
+  JsonSchema,
+} from '@jsonforms/core';
+import { JsonFormsDispatch } from '@jsonforms/react';
+import { Hidden } from '@mui/material';
+
+export type CategorizationElement = Category | Categorization;
+
+export interface StepProps {
+  category: CategorizationElement;
+  categoryIndex: number;
+  step: number;
+  schema: JsonSchema;
+  enabled: boolean;
+  visible: boolean;
+  path: string;
+  disabledCategoryMap: boolean[];
+  renderers: JsonFormsRendererRegistryEntry[] | undefined;
+  cells: JsonFormsCellRendererRegistryEntry[] | undefined;
+}
+
+export const RenderStepElements = (props: StepProps): JSX.Element => {
+  return (
+    /*
+      [Mar-04-2024][Paul Li] the GoAPages internal state cannot handle the hidden/display well. We need extra hide/display control to it appropriately.
+     */
+    <Hidden xsUp={props.categoryIndex !== props.step - 1}>
```

Comment:
Hidden will be replaced by Visible. wait Paul pr merged, then you can address it.